pkg/controller/grouping/convert.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 Antrea Authors
```

Comment:
Sorry, I just noticed this location of this file. I fell it might be a little confusing for developers that the package contains unrelated two topics: one is the calculation of the group members, one is the convertion of ClusterGroup API. What do you think about pkg/apiserver/handlers/webhook/ for this file?

Reply:
I moved it to `pkg/controller/networkpolicy/convert.go` to follow what the validation and mutation webhook's pattern. I'll leave it out of `pkg/apiserver/handlers/webhook/` so that this pkg can focus on the webhook infra and not the actual conversion/mutation/validation logic